src/trace-api.js

```diff
@@ -194,9 +193,10 @@ PluginAPI.prototype.createChildSpan = function(options) {
  * that are called asynchronously (for example, in a network response handler).
  * @param {function} fn A function to which to bind the trace context.
  */
-PluginAPI.prototype.wrap = function(fn) {
+TraceApiImplementation.prototype.wrap = function(fn) {
   if (!this.agent_.namespace) {
-    this.logger_.warn('Trace agent: No CLS namespace to bind function to');
+    this.logger_.warn(this.pluginName_ + ': No CLS namespace to bind ' +
```

Comment:
I know this is a copy but could we fix the grammar. Maybe "No CLS namespace available to bind function"?